src/components/utils/src/file_system_qt.cc

```diff
@@ -172,26 +172,26 @@ bool file_system::DeleteFile(const std::string& name) {
   return false;
 }
 
-void file_system::remove_directory_content(const std::string& directory_name) {
-  QDir dir(QString(directory_name.c_str()));
+void file_system::RemoveDirectoryContent(const std::string& directory_path) {
+  QDir dir(QString(directory_path.c_str()));
   dir.setNameFilters(QStringList() << "*.*");
   dir.setFilter(QDir::Files);
   foreach(QString dirFile, dir.entryList()){
     dir.remove(dirFile);
```

Comment:
@anosach-luxoft, remove errors.
According to Qt documentation ``remove`` can return false.